flair/file_utils.py

```diff
@@ -22,6 +23,20 @@
 CACHE_ROOT = os.path.expanduser(os.path.join('~', '.flair'))
 
 
+def load_big_file(f):
```

Comment:
This code belongs to PR https://github.com/zalandoresearch/flair/pull/397. I guess you added it incidentally?

Reply:
oh oops yes

Reply:
I'll correct and do another PR

Reply:
Great @alanakbik  @tabergma ! 
Thanks a lot for adding it! 
Glad to contribute a little bit after so much help from you guys! Thank you!